db/schema.rb

```diff
@@ -337,15 +245,15 @@
     t.float    "tuition_in_state"
     t.float    "tuition_out_of_state"
     t.float    "books"
```

Comment:
Especially here, are these truly float values or are you just making them float to match the other tables?